### PR TITLE
Finalscreen Copy and Embed

### PR DESCRIPTION
**Pull request by simaonogueira101**

### CODE REVIEW

File: app/components/elements/button-farcaster.tsx [hunk 1]
```diff
@@ -11,9 +11,11 @@ export default function ButtonFarcaster() {
   const { user } = useGetUser();
 
   const shareOnFarcaster = useCallback(() => {
-    const text = encodeURIComponent("Check out my Onchain Wrapped 2024 by Talent Protocol!");
+    const text = encodeURIComponent(
+      `@talentmate.eth Check out my Onchain Wrapped 2024 by Talent Protocol!\nhttps://www.builderscore.xyz/wrapped/${user?.id}/talent`
```
Comment (RubenSousaDinis) [inline app/components/elements/button-farcaster.tsx]:
```suggestion
      `@talentmate.eth Check out my Onchain Wrapped 2024 by Talent Protocol!\nhttps://www.builderscore.xyz/wrapped/${user?.talent_id}/talent`
```

File: app/components/elements/button-farcaster.tsx [hunk 2]
```diff
@@ -11,9 +11,11 @@ export default function ButtonFarcaster() {
...
+    );
     const url = encodeURIComponent(`https://www.builderscore.xyz/wrapped/${user?.id}/talent`);
```
Comment (RubenSousaDinis) [inline app/components/elements/button-farcaster.tsx]:
```suggestion
    const url = encodeURIComponent(`https://www.builderscore.xyz/wrapped/${user?.talent_id}/talent`);
```

**Review by RubenSousaDinis — COMMENTED** [review]
Small fix required

File: app/components/elements/button-farcaster.tsx (hunk 2 above)
Comment (simaonogueira101) [reply app/components/elements/button-farcaster.tsx]: Done 👍

**Review by RubenSousaDinis — APPROVED** [review]

**Pull request merged**